examples/step-70/step-70.cc

```diff
@@ -1095,11 +1095,12 @@ namespace Step70
     // functions will receive a signal when refinement is about to happen, and
     // when it has just happened, and will take care of transferring all
     // information to the newly refined grid with minimal computational cost.
-    fluid_tria.signals.pre_distributed_refinement.connect(
-      [&]() { tracer_particle_handler.register_store_callback_function(); });
+    fluid_tria.signals.pre_distributed_refinement.connect([&]() {
+      tracer_particle_handler.prepare_for_coarsening_and_refinement();
+    });
 
     fluid_tria.signals.post_distributed_refinement.connect([&]() {
-      tracer_particle_handler.register_load_callback_function(false);
+      tracer_particle_handler.transfer_after_coarsening_and_refinement();
     });
```

Comment:
I think it would make sense to call these functions embedded around `fluid_tria.execute_coarsening_and_refinement()` similar to `step-68`, rather than connecting them to signals.